client/my-sites/migrate/controller.js

```diff
@@ -0,0 +1,15 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal Dependencies
+ */
+import SectionMigrate from 'my-sites/migrate/section-migrate';
+
+export function migrateSite( context, next ) {
+	context.primary = <SectionMigrate />;
```

Comment:
Can we put this behind the feature gate as well? Or, do check when rendering the section and return null or something.

Currently, if you go directly to `/migrate` you see the screen, even without the flag enabled.